Concurrency/05select.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+// fmtパッケージを使用するためにImportする
+import "fmt"
+
+// fibonacci関数の定義 int型のチャネル c を引数としてとる
+func fibonacci(c, quit chan int) {
+	x, y := 0, 1
+	for {
```

Comment:
ちなみに、ここのforはこうも書けます。
回数を制御したい場合などに便利かと思います。
```suggestion
	for i := 0; i < 11; i++ {
```

並列処理は色んなパターンあるので、こんな書き方もあるんだなぐらいの参考です。